docs/backhandler.md

```diff
@@ -152,69 +155,16 @@ export default App;
 ```
 
 </TabItem>
-</Tabs>
-
-`BackHandler.addEventListener` creates an event listener & returns a `NativeEventSubscription` object which should be cleared using `NativeEventSubscription.remove` method.
-
-Additionally `BackHandler.removeEventListener` can also be used to clear the event listener. Ensure the callback has the reference to the same function used in the `addEventListener` call as shown the following example ﹣
```

Comment:
This method of removing event handler subscriptions has been deprecated for a while and we shouldn't recommend it as an alternative to the right way to.